### ISSUE TITLE
Update documentation job-yaml/json parameters

### ISSUE THREAD

**Issue by AWKIF** [rundeck/rundeck#5357]
**Describe the bug**

Not really a bug but there is missing parameters from the online documentation regarding jobs definition:
https://docs.rundeck.com/docs/manual/document-format-reference/job-yaml-v12.html
https://github.com/rundeck/docs/blob/3.1.x/docs/manual/document-format-reference/job-yaml-v12.md

Exporting jobs in yaml from the UI we noticed these parameters, it's kind of obvious what they do but can you add them in the documentation as our teams define jobs directly in yaml in github and our CI auto import them in rundeck
```
nodeFilterEditable
notifyAvgDurationThreshold
nodesSelectedByDefault
orchestrator

```

TL;DR: all parameters/options available in the UI should be described in the job-yaml/json definition documentation

Thanks a lot
